alacritty/src/logging.rs

```diff
@@ -107,12 +107,15 @@ impl log::Log for Logger {
             return;
         }
 
-        let now = time::strftime("%F %T.%f", &time::now()).unwrap();
-        let msg = format!("[{}] [{:<5}] [{}] {}\n", now, record.level(), target, record.args());
+        // Create log message for the given `record` and `target`.
+        let msg = create_log_message(record, &target);
+
+        // Trim extra spaces if there're any.
+        let msg = msg.trim_end_matches(' ').as_ref();
```

Comment:
Why is this not part of `create_log_message`? Because it returns a reference? Could probably just convert it to owned or something?

Reply:
Yeah, but it's one more allocation for nothing, so I've decided to do that separately.

Reply:
I mean if that's the only concern, maybe use `rposition` and `truncate`?

Reply:
I think just `truncate` should be fine?